### ISSUE TITLE
JSON signals import file

### ISSUE THREAD

**Issue by ng-designs** [akumidv/tradingview-assistant-chrome-extension#147]
Добрый день. А можно добавить возможность загружать внешние сигналы из JSON файла? И есть ли пример добавления осциллятора с загрузкой внешних данных или это невозможно?

**Comment by akumidv** [comment]
Yes it possible add from JSON, but the problem there than there need to set schema of this JSON and it harder to convert JSON from one JSON schema to another one instead of converting to CSV. So I didn't see advantage of using JSON. 

About second part of your question. This is need to implement this oscillator as pine script and update extension code to upload signals.

For now I'm thinking about adding external trades to chart, probably with equity too (probably as an strategy).
